File: app/src/main/java/com/example/dcct/view/QueryCallback.java
package com.example.dcct.view;

import com.example.dcct.base.BaseCallback;
import com.example.dcct.bean.BackResultData;
import com.example.dcct.bean.QueryResultEntity;

import java.util.List;

public interface QueryCallback extends BaseCallback {
    void onLoadQueryData(BackResultData<List<QueryResultEntity>> backResultData);
}

File: app/src/main/java/com/example/dcct/view/QueryCallback.kt
package com.example.dcct.view

import com.example.dcct.base.BaseCallback
import com.example.dcct.bean.BackResultData
import com.example.dcct.bean.QueryResultEntity

interface QueryCallback : BaseCallback {
    fun onLoadQueryData(backResultData: BackResultData<List<QueryResultEntity>>)
}